### PR TITLE
Remove other relief data links

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: source/dataset/earth-relief.rst [hunk 1]
```diff
@@ -102,8 +102,6 @@ earth_relief: 全球地形起伏数据
 其它机构或单位也提供了不同格式、不同分辨率的地形起伏数据。此处仅列出部分，
 供需要的用户参考与使用：
 
-- ETOPO1 (1弧分): https://www.ngdc.noaa.gov/mgg/global/
-- GEBCO (30弧秒): https://www.gebco.net/
-- SRTM30+ (30弧秒): https://topex.ucsd.edu/WWW_html/srtm30_plus.html
-- SRTM3 (3弧秒，仅陆地): http://srtm.csi.cgiar.org
```
Comment (core-man) [inline source/dataset/earth-relief.rst:108]: This link is bad and we already list **SRTMGL1** and **SRTMGL3** above.

> 数据来源及引用
#. SRTM15+: http://dx.doi.org/10.1029/2019EA000658
#. SRTMGL3数据: https://lpdaac.usgs.gov/products/srtmgl3v003/
#. SRTMGL1数据: https://lpdaac.usgs.gov/products/srtmgl1v003/

File: source/dataset/earth-relief.rst [hunk 2]
```diff
@@ -102,8 +102,6 @@ earth_relief: 全球地形起伏数据
 其它机构或单位也提供了不同格式、不同分辨率的地形起伏数据。此处仅列出部分，
 供需要的用户参考与使用：
 
-- ETOPO1 (1弧分): https://www.ngdc.noaa.gov/mgg/global/
-- GEBCO (30弧秒): https://www.gebco.net/
-- SRTM30+ (30弧秒): https://topex.ucsd.edu/WWW_html/srtm30_plus.html
```
Comment (core-man) [inline source/dataset/earth-relief.rst:107]: SRTM30+ is old and we have SRTM15+ listed above.

File: source/dataset/earth-relief.rst [hunk 3]
```diff
@@ -102,8 +102,6 @@ earth_relief: 全球地形起伏数据
 其它机构或单位也提供了不同格式、不同分辨率的地形起伏数据。此处仅列出部分，
 供需要的用户参考与使用：
 
-- ETOPO1 (1弧分): https://www.ngdc.noaa.gov/mgg/global/
-- GEBCO (30弧秒): https://www.gebco.net/
```
Comment (core-man) [inline source/dataset/earth-relief.rst:106]: Shall we just remove them all?

Comment (seisman) [reply source/dataset/earth-relief.rst:106]: Yes to me

Comment (core-man) [reply source/dataset/earth-relief.rst:106]: Done in 10e027c

**Review by seisman — APPROVED** [review]

**Pull request merged**